src/main/java/domain/Players.java

```diff
@@ -34,4 +46,8 @@ public int getSize() {
     public List<String> getNames() {
         return players.stream().map(Player::getName).toList();
     }
```

Comment:
Stream 을 사용하실 땐 . 하나당 한줄 씩 띄워주면 읽기 편해집니다.
```suggestion
    public List<String> getNames() {
        return players.stream()
                .map(Player::getName)
                .toList();
    }
```

Reply:
저는 너무 긴 경우만 구분해주었는데,
확실히 비교해놓고 보니깐 stream의 흐름이 한 눈에 들어오는 것 같아요. 수정하겠습니다.!